compiler/test/stdlib/list.test.gr

```diff
@@ -219,3 +220,24 @@ assert init(-10, idx => fail "Shouldn't be called") == []
 // List.join
 assert join(", ", ["a", "b", "c"]) == "a, b, c"
 assert join(", ", []) == ""
+
+
+// List.sort
+let comp = (left, right) => {
+    match((left, right)) {
+        (left, right) when left > right => 1,
+        (left, right) when left == right => 0,
+        _ => -1
+    }
+}
+assert sort(comp, [3,5,2,4,1]) == [1,2,3,4,5]
+assert sort(comp, []) == []
+let comp = (left, right) => {
+    match((String.length(left), String.length(right))) {
+        (left, right) when left > right => 1,
+        (left, right) when left == right => 0,
+        _ => -1
+    }
+}
+let list = ["a", "abcde", "abc", "ab", "abcd", "a"]
+assert sort(comp, list) == ["a", "a", "ab", "abc", "abcd", "abcde"]
```

Comment:
Looks good!  Just nitpicking, Could we test a couple of edge cases - an empty list, a list with values all the same etc?

Reply:
What do you have in mind?  There is an empty list test on line 234 and this test has duplicate items in it ("a").  Will be happy to add more if you have a specific case in mind

Reply:
I'll add 1 more with all identical items